src/main/java/frc/robot/commands/manipulator/commandgroups/AutoFirstShot.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.manipulator.commandgroups;
```

Comment:
what is the difference between commandgroups and independantgroups? Plus independent is misspelled.

Reply:
just rename from:
1. `commandgroups` -> `commandgroup`
2. `independantgroups` -> `independentgroup`